README.md

```diff
@@ -122,36 +122,29 @@ A pre-built container image of the shopping cart service is provided as `lightbe
 If you have built your own container image, change the image in the following command to point to the one that you just pushed.
 
 ```shell
-$ csctl svc deploy shopping-cart \
+$ akkasls svc deploy shopping-cart \
     lightbend-docker-registry.bintray.io/cloudstate-samples/shopping-cart-java
 ```
 
 Wait for the shopping cart service `STATUS` to be `ready`.
 
 ```shell
-$ csctl svc get
+$ akkasls svc get
 ```
 
-### Expose the frontend service
-
-```shell
-$ csctl svc expose frontend
-```
+### Frontend service
 
-The output will look like this:
+The `frontend` service is a frontend web application written in TypeScript.
+It is backed by a `stateless` service that will serve the compiled JavaScript, html and images. This service makes `grpc-web` calls directly to the other services to get the data that it needs.
```

Comment:
* Do you want "stateless" in a fixed mono font or italics?  Italics seems to make more sense to me here.
* I'd also capitalize HTML.

Reply:
```suggestion
It is backed by a _stateless_ service that will serve the compiled JavaScript, HTML and images. This service makes `grpc-web` calls directly to the other services to get the data that it needs.
```